### DISCUSSION TITLE
3.14.5 problems

### DISCUSSION THREAD

**Discussion by TubaOrNotTuba** [dhowe/AdNauseam#2176] [Q&A]
When I launch Edge, every site that I have set to launch on startup says blocked then a few seconds later load.  This just started happening.  If I disable AdNauseam the behaviour stops.  How can I prevent this from continuing?

**Comment by dhowe** [comment]
@mneunomne can you check this?

**Comment by mneunomne** [comment]
I wasn't able to reproduce this on edge at the moment. @TubaOrNotTuba could you please inspect in the extension manager to see if there are any error messages showing up?
